src/views/v2/rsvp/form/fields.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Block: RSVP
+ * Form fields
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/form/fields.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+$this->template( 'v2/rsvp/form/fields/name', [ 'rsvp' => $rsvp, 'going' => $going ] );
+$this->template( 'v2/rsvp/form/fields/email', [ 'rsvp' => $rsvp, 'going' => $going ] );
+$this->template( 'v2/rsvp/form/fields/quantity', [ 'rsvp' => $rsvp, 'going' => $going ] );
```

Comment:
why not just add these directly to `src/views/v2/rsvp/form/fields`?

Reply:
You mean `src/views/v2/rsvp/form/form.php`? I thought it was cleaner to make it this way. 

Plus, we give the ability to have the template hook before and after the fields, independently of how many we add/remove here. If somebody wants to add more fields, they don't have to do it after an specific field, but they can do it at the beginning or the end, without even having to worry if we put more, or remove some of these.